api/openapi.yaml

```diff
@@ -822,13 +822,21 @@ paths:
           $ref: "#/components/responses/401"
         403:
           $ref: "#/components/responses/403"
-  /paths/{pathId}:
+  /paths/{userId}/{pathId}:
```

Comment:
I would suggest not to provide userId in the path. Since park/path is a separate resource that can exist without a user. Just compare the authenticated user with the creator of the park to be deleted. It will also saves us some time for checks and tests.

Reply:
updated as suggested